appvm-scripts/usr/lib/qubes/qubes-keymap.sh

```diff
@@ -30,10 +27,7 @@ set_keyboard_layout() {
     done
 }
 
-if [ -n "$QUBES_KEYMAP" ]; then
-    set_keyboard_layout "$QUBES_KEYMAP"
-fi
```

Comment:
You need to leave at least this one (and corresponding qubesdb-read), to load initial keyboard layout.